rotkehlchen/db/accounting_rules.py

```diff
@@ -0,0 +1,140 @@
+from typing import TYPE_CHECKING, Optional
+
+from pysqlcipher3 import dbapi2 as sqlcipher
+
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.evm.accounting.structures import BaseEventSettings
+from rotkehlchen.db.constants import NULL_ACCOUNTING_COUNTERPARTY
+from rotkehlchen.db.filtering import AccountingRulesFilterQuery
+from rotkehlchen.errors.misc import InputError
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+
+
+class DBAccountingRules:
+
+    def __init__(self, db_handler: 'DBHandler') -> None:
+        self.db = db_handler
+
+    @classmethod
+    def _formulate_primary_key(
+            cls,
+            event_type: HistoryEventType,
+            event_subtype: HistoryEventSubType,
+            counterparty: Optional[str],
+    ) -> tuple[str, str, str]:
```

Comment:
1. Add docstring.
2. Is this really needed to be its own method? It's really realy simple logic which can just be added in the 2 places you use it. Not a dealbreaker for me ofc, but just looks weird.